src/components/MaterialUI/Inputs/Select.js

```diff
@@ -93,26 +78,28 @@ const Select = ({ options, selectProps }) => {
 
 	const update = selectProps?.get(SelectProps.propNames.update);
 	const value = selectProps?.get(SelectProps.propNames.value);
-	const numericSort = selectProps?.get(SelectProps.propNames.numericSort) || false;
-	const showAllValue = selectProps?.get(SelectProps.propNames.showAllValue) || "#All#";
-	const showAllLabel = selectProps?.get(SelectProps.propNames.showAllLabel) || "Show All";
+	const sortType = selectProps?.get(SelectProps.propNames.sortType) || sortTypeEnum.none;
+	const showAllValue = selectProps?.get(SelectProps.propNames.showAllValue);
+	const showAllLabel = selectProps?.get(SelectProps.propNames.showAllLabel);
+	const classesProps = selectProps?.get(SelectProps.propNames.classes);
```

Comment:
Classes shouldn't be defined through props. The rule names should be defined through componentClasses and then use getStyle and setStyle. So, for this case ruleNames could be menuPropsPaper and root. Check dividerProps for sample